packages/x-codemod/README.md

```diff
@@ -108,33 +108,33 @@ This change only affects Date and Time Picker components.
 ```
 
 ```bash
-npx @mui/x-codemod v7.0.0/pickers/rename-components-to-slots <path>
+npx @mui/x-codemod@next v7.0.0/pickers/rename-components-to-slots <path>
 ```
 
 #### `rename-default-calendar-month-to-reference-date`
 
 Replace the `defaultCalendarMonth` prop with the `referenceDate` prop.
 
 ```diff
-- <DateCalendar defaultCalendarMonth={dayjs('2022-04-01')};
-+ <DateCalendar referenceDate{dayjs('2022-04-01')} />
+-<DateCalendar defaultCalendarMonth={dayjs('2022-04-01')};
++<DateCalendar referenceDate{dayjs('2022-04-01')} />
```

Comment:
The format with the space looks weird. Same reason why we usually write:

```jsx
console.log(foo);
```

and not:

```jsx
 console.log(foo);
```